### PR TITLE
Classifier

### PR DESCRIPTION
**Pull request by corochann**
I want to introduce Classifier & Regressor wrapper class

The main motivation is to support 
1. `predict`, `predict_proba` (classifier only) method to provide much easier API at inference stage.
2. `accuracy` key name can be configured (related to issue #115 )

This first PR only contains implementation for Classifier.

### CODE REVIEW

**Comment by corochann** [comment]
I updated code, please review.

Its usage is written in example as another PR https://github.com/pfnet-research/chainer-chemistry/pull/125 (example code itself will be updated later.)
https://github.com/corochann/chainer-chemistry/blob/classifier_example/examples/tox21/predict_tox21.py#L116-L119

File: chainer_chemistry/models/prediction/classifier.py [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+    """A simple classifier model.
+
+    This is an example of chain that wraps another chain. It computes the
+    loss and accuracy based on a given input/label pair.
+
+    Args:
+        predictor (~chainer.Link): Predictor network.
+        lossfun (function): Loss function.
+        accfun (function): DEPRECATED. Please use `metrics_fun` instead.
+        metrics_fun (function or dict or None): Function that computes metrics.
```
Comment (mottodora) [inline chainer_chemistry/models/prediction/classifier.py]: I feel `metrics` is enough. How do you think?

File: chainer_chemistry/models/prediction/classifier.py [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+          >>> def lossfun(x, t):
+          ...     return F.softmax_cross_entropy(
+          ...         x, t, enable_double_backprop=True)
+          >>>
+          >>> predictor = L.Linear(10)
+          >>> model = L.Classifier(predictor, lossfun=lossfun)
+
+    """
+
+    compute_metrics = True
```
Comment (mottodora) [inline chainer_chemistry/models/prediction/classifier.py]: Do you use this `compute_metrics`?

File: examples/tox21/predict_tox21.py [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+from rdkit import RDLogger
+import six
+
+from chainer_chemistry import datasets as D
+
+import data
+import predictor
+
+
+# Disable errors by RDKit occurred in preprocessing Tox21 dataset.
```
Comment (mottodora) [inline examples/tox21/predict_tox21.py]: Wrong place?
